rules/windows/command_and_control_common_webservices.toml

```
@@ -128,10 +127,18 @@ This rule looks for processes outside known legitimate program locations communi
 - Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
 - Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
 """
+references = ["https://www.elastic.co/security-labs/operation-bleeding-bear"]
```

siestagraph uses a websvc domain for C2 `graph.microsoft.com` (covered in this rule)

```suggestion
references = [
"https://www.elastic.co/security-labs/operation-bleeding-bear", 
"https://www.elastic.co/security-labs/siestagraph-new-implant-uncovered-in-asean-member-foreign-ministry"
]
```